tests/analysis/trials/test_has_passed_step_scenarios.py

```diff
@@ -20,7 +20,7 @@ def test_single_writer_all_steps_written_complete_job():
     END_OF_JOB.ts --> Present
     """
 
-    path = "s3://tornasole-testing/has_step_scenarios/single-writer-all-steps-written-complete-job"
+    path = "s3://smdebug-testing/has_step_scenarios/single-writer-all-steps-written-complete-job"
```

Comment:
I believe these paths have some files needed for the tests. Have you copied them over to new location?

Reply:
If you have not already, can you put them in a resources directory, so we know that we can clean everything except resources

Reply:
Done. We should really move away from relying on S3 state for tests, spoke with Nihal about it.